### PR TITLE
Adapt the rest wake lock tests to latest spec

### PR DESCRIPTION
**Pull request by Honry**
- Remove some tests since `active`, `type` and
  `onactivechange` had been removed from latest spec
- Enable insecure context and invaild type tests on Worker

### CODE REVIEW

**Comment by Honry** [comment]
@rakuco, @kenchris, could you PTAL?

**Review by kenchris — APPROVED** [review]

File: wake-lock/wakelock-type.https.any.js
Comment (rakuco) [inline wake-lock/wakelock-type.https.any.js]: You need to make this function `async` in order to use `await` in the line below.

Comment (Honry) [reply wake-lock/wakelock-type.https.any.js]: Done, thanks!

File: wake-lock/wakelock-type.https.any.js
Comment (rakuco) [inline wake-lock/wakelock-type.https.any.js]: You can drop `async` here since it's only needed in line 16.

**Review by rakuco — APPROVED** [review]
Thanks, I thought I'd had fixed all tests in my Chromium CL but overlooked quite a lot of them :-) I've added an inline comment, feel free to merge after fixing that one if all bots pass (ie. there's no need to wait for another review).

File: wake-lock/wakelock-type.https.any.js
Comment (Honry) [reply wake-lock/wakelock-type.https.any.js]: Ops... Done!

**Comment by Honry** [comment]
@rakuco, @kenchris, many thanks for your review! I'm merging it.

**Pull request merged**
